### PR TITLE
👷 Handle symbolic links between host platforms and containers

### PR DESCRIPTION
**Pull request by AlexV525**
_Description of what this PR is changing or adding, and why:_
Replaces all symlinks to the actual destination by copying corresponding files and directories.

_Issues fixed by this PR (if any):_
- https://github.com/flutter/website/issues/6201
- https://github.com/flutter/website/issues/6443
- https://github.com/flutter/website/issues/8272

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by AlexV525** [comment]
Requires #9307.

**Comment by AlexV525** [comment]
On Windows 🎆

image

File: examples/get-started/codelab_web/lib/starter.dart [hunk 1]
```diff
@@ -47,7 +47,7 @@ class _SignUpFormState extends State<SignUpForm> {
   final _lastNameTextController = TextEditingController();
   final _usernameTextController = TextEditingController();
 
-  double _formProgress = 0;
+  final double _formProgress = 0;
```
Comment (domesticmouse) [inline examples/get-started/codelab_web/lib/starter.dart:50]: I'm curious why this is final. Going by the name, `_formProgress` feels like it should be updated during the progress of the form?

Comment (AlexV525) [reply examples/get-started/codelab_web/lib/starter.dart:50]: Great observation. I modified it because the linter raises warnings, I'll take another look.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
PTAL @parlough

**Comment by AlexV525** [comment]
@parlough It's not working. :(

**Comment by parlough** [comment]
I think that's just because the test.sh script isn't updated to handle the changes. That's why the other PR changes it.

**Comment by atsansone** [comment]
@AlexV525 , @parlough : Is this still an active concern or fix? If so, I'd love to approve and merge soon.

**Comment by AlexV525** [comment]
> I think that's just because the test.sh script wasn't updated to handle the changes. That's why the other PR simplifies the testing.

It's ready on my side. Passing to @parlough. :)

**Comment by atsansone** [comment]
@parlough : Can you make the needed changes please?

**Comment by parlough** [comment]
Yep. I discussed with Alex and I'll get back to this soon if something else I'm trying doesn't work out.

**Comment by parlough** [comment]
Thanks for working on this Alex and for being patient!! Sorry about the recurring delay on reviewing it and getting it landed.

We plan to remove the Docker setup from the repository, at least for now, sometime within the next few weeks. I'll have more updates soon. I'll also be making sure to spend time getting things working on Windows too.

So since this is going to be dropped since, I'll close this for now. Thanks again 💙

**Comment by AlexV525** [comment]
> We plan to remove the Docker setup from the repository, at least for now, sometime within the next few weeks

Great move, congrats!

**Pull request closed without merging**